internal/venice-common/src/main/java/com/linkedin/venice/serialization/KeyWithChunkingSuffixSerializer.java

```diff
@@ -53,6 +53,13 @@ public ByteBuffer serializeNonChunkedKey(ByteBuffer key) {
     return serialize(key, serializedNonChunkKeySuffix);
   }
 
+  public byte[] getKeyFromChunkedKey(byte[] keyBytesWithChunkSuffix) {
```

Comment:
The assumption here is that the chunked key is only for the whole value, and it doesn't handle the value chunks.
How can we guarantee the caller always invokes this method with chunked key for the whole value entry?

Reply:
We cannot, that's why it's not a proper fix like I mentioned in the comments. Just a tactical fix for now that will allow stores that require chunking enabled (read compute) but doesn't actually have any large values. Meanwhile, we will work on proper chunking support in another PR.

Reply:
Got it.